### PR TITLE
templates: numbers of users in authorise page

### LINKED ISSUE #159: templates: authorise page should show number of users and the creation date
The authorise page should show how many users are given application have, and how long ago the application was created (in order allow users to determine if it is a fake app).

### PR DESCRIPTION
**Pull request by MiltiadisAlexis**
* Add number of users ( closes #159)

Signed-off-by: Miltiadis Alexis miltiadis.alexis@cern.ch

### CODE REVIEW

**Comment by MiltiadisAlexis** [comment]
image

**Comment by MiltiadisAlexis** [comment]
image

File: invenio_oauth2server/models.py [hunk 1]
```diff
@@ -289,6 +289,15 @@ def reset_client_secret(self):
             current_app.config.get('OAUTH2SERVER_CLIENT_SECRET_SALT_LEN')
         )
 
+    @property
+    def get_users(self):
+        """Get number of users."""
+        no_users = Token.query.filter_by(
+            client_id=self.client_id,
+            is_personal=False
```
Comment (drjova) [inline invenio_oauth2server/models.py]: I would say to add ``is_internal=False`` as well :)

**Comment by MiltiadisAlexis** [comment]
image

File: invenio_oauth2server/templates/invenio_oauth2server/authorize.html [hunk 2]
```diff
@@ -54,6 +54,9 @@ <h3 align="center">{{ _("Application '%(client_name)s' by '%(client_user)s' want
     <h4>{{client.name}}</h4>
     {%- if client.description %}<p>{{client.description}}</p>{% endif %}
     {%- if client.website %}<p><a href="{{client.website}}">{{ _('Visit application website') }}</a></p>{% endif %}
+    <hr />
+    <p class="text-center text-muted">
+    <i class="fa fa-users"></i> {{ client.get_users }} {{ 'user' if client.get_users == 1 else 'users' }}
```
Comment (drjova) [inline invenio_oauth2server/templates/invenio_oauth2server/authorize.html]: ``</p>`` is missing :P

**Pull request merged**